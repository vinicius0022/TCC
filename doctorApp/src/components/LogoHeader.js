import React, { Component } from "react";
import { StyleSheet, View, Text, Dimensions, Image } from "react-native";

class LogoHeader extends Component {

  render() {

    return (
      <View style={styles.header}>
        <Image style={styles.logo} source={require("../../assets/images/icon.png")} />
        <View style={styles.flexFiller}></View>
        <View style={styles.text3Column}>
          <Text style={styles.text3}>Doctor App</Text>
          <View style={styles.rect7}></View>
        </View>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  header:{
    width: '100%',
    height:'25%',
    alignItems: 'center',
    justifyContent: 'center',
    marginTop: 10,
  },
  logo: {
    width: Dimensions.get('window').width / 4,
    height: Dimensions.get('window').width / 4,
    alignSelf: "center"
  },
  text3: {
    color: "rgba(255,255,255,255)",
    fontSize: 20,
  },
});

export default LogoHeader;
