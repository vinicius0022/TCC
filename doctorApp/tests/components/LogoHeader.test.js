import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'
import LogoHeader from '../../src/components/LogoHeader';


describe('render', ()=>{

    it('show LogoHeader', () =>{

        const shallowRenderer = new ShallowRenderer();
        shallowRenderer.render(<LogoHeader/>)

        const result = shallowRenderer.getRenderOutput();
        expect(result).toMatchSnapshot();
    })
})