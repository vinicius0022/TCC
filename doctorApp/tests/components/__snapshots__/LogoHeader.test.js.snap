// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render show LogoHeader 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "height": "25%",
      "justifyContent": "center",
      "marginTop": 10,
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../assets/images/icon.png",
      }
    }
    style={
      Object {
        "alignSelf": "center",
        "height": 187.5,
        "width": 187.5,
      }
    }
  />
  <View />
  <View>
    <Text
      style={
        Object {
          "color": "rgba(255,255,255,255)",
          "fontSize": 20,
        }
      }
    >
      Doctor App
    </Text>
    <View />
  </View>
</View>
`;
